internal/report/report.go

```diff
@@ -89,6 +91,10 @@ func (r *Reporter) SendReport(ctx context.Context) (*DataResponse, error) {
 	rCtx, cancel := context.WithTimeout(ctx, 5*time.Second)
 	defer cancel()
 
+	var m runtime.MemStats
```

Comment:
It feels a tad sneaky do this in "SendReport". At least we should add that this happens to the functions comment, if we don't find a better place to put this in.

Reply:
Updated the function comment. `SendReport` is periodically called by the runtime. So the report will contain the latest memory usage of the instance.